.storybook/preview.js

```diff
@@ -7,7 +7,13 @@ export const decorators = [withFluentProvider, withStrictMode];
 /** @type {import('@storybook/react').Parameters} */
 export const parameters = {
   controls: {
-    disable: true,
+    hideNoControlsWarning: true,
     expanded: true,
   },
+  viewMode: 'docs',
```

Comment:
**chore:** based on discussion I dont think this should be default. As a developer working with SB as primary evnironment one uses `canvas` primarily. this should be tweaked only for specific builds for public consumption like react-component

Reply:
also for "public docs" output it would be probably good idea to use `docs` mode when building 
-> https://storybook.js.org/docs/react/writing-docs/build-documentation#publish-storybooks-documentation